srcpkgs/stcgal/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'stcgal'
+pkgname=stcgal
+version=1.6
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="stcgal"
+hostmakedepends="python3-setuptools"
+depends="python3-pyserial python3-setuptools python3-tqdm"
+short_desc="STC MCU ISP flash tool"
+maintainer="bra1nwave <brainwave@openmailbox.org>"
+license="MIT"
+homepage="https://github.com/grigorig/stcgal"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=1a526503226886a6c4d7fb8296e5530053bcac63fe6f4a29d5f6c40f86f9e20f
+
+post_install() {
+	vlicense ${FILESDIR}/LICENSE
```

Comment:
i'm not keen on vendoring the license, can't you put the link on the distfiles ?

Reply:
Sadly, there is no license file in that repo. 
I could take the license from the [AUR](https://aur.archlinux.org/cgit/aur.git/plain/LICENSE?h=stcgal) (if that's allowed, of course, which I doubt).

Reply:
@bra1nwave Can't you extract the License out of the Source files? Like `head -n21 stcgal/frontend.py > LICENSE`

Reply:
Good idea, Johnnynator. Thanks.